src/workflow.proto

```diff
@@ -30,6 +30,10 @@ message DateTimeParameter {
   optional string default = 1;
 }
 
+message DurationParameter {
+  optional int64 default = 1;
```

Comment:
If it is for time duration, could it be `uint` instead of `int`? (I was looking at `job.proto` where `timeout_ms` is defined as `uint32` type)

Reply:
Thank you, good catch!

Reply:
Fixed with https://github.com/Project-OMOTES/omotes-sdk-protocol/pull/16/commits/b4c62a2ed21d40089f2c7b105744452a02335dc1